mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```
@@ -110,6 +110,8 @@ storage:
           # Start by default on boot
           WantedBy=multi-user.target
           [Service]
+          # Extend start time because we are pulling from quay and doing an install
```

Note that the service is marked as started when the container starts ( except on oneshot units) 
Also, this does not do an install, it boots a VM with netboot, which boot of the iSCSI target, on which coreOS was installed in the step before.

https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#notify-defaults-to-false

 ```suggestion
          # Extend start time because we are pulling from quay and cosa is a large image
```

---

Sounds good. I'll make the change and then merge once you approve (skipping CI cycle) since it's only a comment change.

---

Updated!